Update interfaces/scroll-to-text-fragment.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Note: This file was recently manually updated in commit cd5f4dfe12.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/21d7c4a/ed/idl/scroll-to-text-fragment.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/721189151

foolip commented on interfaces/scroll-to-text-fragment.idl:
```diff
@@ -3,10 +3,10 @@
 // Source: Text Fragments (https://wicg.github.io/scroll-to-text-fragment/)
 
-[Exposed=Window]
+[Exposed=Document]
```
This is wrong, will have to go fix the spec.

bokand replied:
Sorry about that, I changed this recently in https://github.com/WICG/scroll-to-text-fragment/pull/134 and it looks like I misunderstood what `Exposed` does. I've changed it back to `Window` which I think is correct (this makes it accessible only from the main thread rather than workers?) and opened a PR; added you as reviewer to make sure I got it right this time.

foolip replied:
That's right, Window effectively means the main thread, the context where the global object is a Window.

autofoolip committed:
Update interfaces/scroll-to-text-fragment.idl

Note: This file was recently manually updated in commit cd5f4dfe12.
      This commit may revert some of those changes.

Source: https://github.com/tidoust/reffy-reports/blob/21d7c4a/ed/idl/scroll-to-text-fragment.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/721189151
